### PR TITLE
add python3-prompt-toolkit to rosdep

### PR DESCRIPTION
**Pull request by ipa-fez**
Please add the following dependency to the rosdep database.

## Package name:

python3-prompt-toolkit

## Package Upstream Source:

https://github.com/prompt-toolkit/python-prompt-toolkit

## Purpose of using this:

prompt_toolkit is a library for building powerful interactive command line and terminal applications in Python.

We're using it to build terminal interfaces to interactively trigger some actions.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/buster/python3-prompt-toolkit
- Ubuntu: https://packages.ubuntu.com/
   - https://packages.ubuntu.com/focal/python3-prompt-toolkit
- Fedora: https://src.fedoraproject.org/browse/projects/
  - https://src.fedoraproject.org/rpms/python-prompt-toolkit
- Arch: https://www.archlinux.org/packages/
  - https://archlinux.org/packages/community/any/python-prompt_toolkit/
- Gentoo: https://packages.gentoo.org/
  - https://packages.gentoo.org/packages/dev-python/prompt_toolkit
- macOS: https://formulae.brew.sh/
  - not available
- Alpine: https://pkgs.alpinelinux.org/packages
  - not available
- NixOS/nixpkgs: https://search.nixos.org/packages
  - not available

### CODE REVIEW

**Review by wjwwood — CHANGES_REQUESTED** [review]
Looks like you could add a key for rhel, according to our checker, can you look into that?

File: rosdep/python.yaml [hunk 1]
```diff
@@ -6953,6 +6953,12 @@ python3-progressbar:
 python3-prometheus-client:
   debian: [python3-prometheus-client]
   ubuntu: [python3-prometheus-client]
+python3-prompt-toolkit:
+  arch: [python-prompt_toolkit]
+  debian: [python3-prompt-toolkit]
+  fedora: [python-prompt-toolkit]
+  gentoo: [dev-python/prompt_toolkit]
+  ubuntu: [python3-prompt-toolkit]
```
Comment (nuclearsandwich) [inline rosdep/python.yaml:6961]:
```suggestion
python3-prompt-toolkit:
  arch: [python-prompt_toolkit]
  debian: [python3-prompt-toolkit]
  fedora: [python-prompt-toolkit]
  gentoo: [dev-python/prompt_toolkit]
  rhel: [python3-prompt-toolkit]
  ubuntu: [python3-prompt-toolkit]
```

**Comment by ipa-fez** [comment]
Looks like it's only available for rhel 8. I tried making it conditional, but the CI still fails and I can't really tell why.

Did mess up the syntax?

**Comment by nuclearsandwich** [comment]
@ipa-fez the error is related to an issue I accidentally introduced into the master branch yesterday and has been resolved by #30010. With maintainer contributions on I merged master into this branch and that should resolve the issue. Sorry for the trouble

**Comment by ipa-fez** [comment]
@wjwwood I've added the rhel key and the pipeline is now green (thanks for the help). Could you please re-review?

**Review by wjwwood — APPROVED** [review]
Lgtm

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**